notebook/services/contents/filemanager.py

```diff
@@ -275,17 +289,17 @@ def _dir_model(self, path, content=True):
                     continue
 
                 try:
-                    st = os.stat(os_path)
+                    st = os.stat(os_path, follow_symlinks=self.follow_symlinks)
                 except OSError as e:
                     # skip over broken symlinks in listing
                     if e.errno == errno.ENOENT:
                         self.log.warning("%s doesn't exist", os_path)
                     else:
-                        self.log.warning("Error stat-ing %s: %s", os_path, e)
+                        self.log.warning("Error stat-ing %s: %s", (os_path, e))
                     continue
 
                 if not stat.S_ISREG(st.st_mode) and not stat.S_ISDIR(st.st_mode):
-                    self.log.debug("%s not a regular file", os_path)
+                    self.log.warning("%s not a regular file", os_path)
```

Comment:
Just made this change to confirm why the mapped volume *still* isn't showing.
Results in warnings being printed to the terminal:
```
[W 17:07:19.450 LabApp] C:\jupyter\notebooks not a regular file
[W 17:07:29.451 LabApp] C:\jupyter\notebooks not a regular file
[W 17:07:40.452 LabApp] C:\jupyter\notebooks not a regular file
```